euth/projects/templates/euth_projects/project_detail.html

```diff
@@ -32,26 +32,30 @@
                 {% if item.type %}
                 <li class="{% if item.id == view.project.active_phase.id %}active{% endif %} timeline-phase">
 
+                  {% if item.id in ids or item.id == view.project.active_phase.id %}
+                  <a href="{% url 'project-detail' slug=view.project.slug %}?phase={{ item.weight }}">
+                  {% endif %}
+
                       <button type="button"
                       class="dot"
                       data-toggle="popover"
                       title="{% trans 'Phase' %}
                         {% if item.id == view.project.active_phase.id %}
                           <div class='label label-secondary label-xs float-right'>{% trans 'Active phase' %}</div>
                         {% endif %}"
-                      data-trigger="hover | click"
+                      data-trigger="hover | focus"
                       data-html="true"
                       data-placement="top"
                       id="button-phase-{{ forloop.counter }}"
                       data-content="
                       <p class='small'>{{ item.start_date|date:"SHORT_DATETIME_FORMAT" }} - {{ item.end_date|date:"SHORT_DATETIME_FORMAT" }}</p>
-                      <h4>{{ item.name }}</h4>
+                      <h4>{{ item.name }}</h4>"
+                      >
```

Comment:
Should this be here?

Reply:
yes, its a thing with popovers, they have the html template within the data-content attribute